### ISSUE TITLE
BUG: Chat bubbles from the other person don't show if you're private chatting with them on Telegram

### ISSUE THREAD

**Issue by fplord** [igneous-labs/omnispace#41]
This is a pretty tough one.

When you're in a private chat with another Telegram user, both users receive each others' replies, but neither user can see each other's message as bubbles. 

E.g. @fp:melchior.info is messaging @lieu:melchior.info on the FP <> Lieu (Telegram) private chat. Messages are successfully exchanged but on fp's side, lieu's incoming messages will not show as bubbles and vice versa.

**why this happens:** Due to how matrix bridges work, you're not actually talking to lieu, you're talking to a Telegram bridge bot, which in FP's case is `@telegram_46353772:melchior.info`, and in lieu's case is `@telegram_235047014:melchior.info`. These telegram bridge bots obviously don't exist in the game world and don't correspond to `@lieu` or `@fp`, so the bubbles don't show up.

@f8122dac91 think need your help here. Is there a mapping between all such telegram bridge bots (there are N for each user) and the user?

**Comment by f8122dac91** [comment]
Yeah this seems to be a quite tough problem to solve. I can't find any easily accessible mapping between these two types of user handle. I'll do some digging in bridge projects themselves
